action.yml

```
@@ -0,0 +1,282 @@
+    required: false
+    default: ""
+  use_docker:
+    description: Whether to use docker to run the server.
+    required: false
+    default: "false"
+  without_movies:
+    description: Whether to create a movies library (new agent).
+    required: false
+    default: "false"
+  without_movies_imdb:
+    description: Whether to create a movies library (IMDB agent).
+    required: false
+    default: "false"
+  without_movies_tmdb:
+    description: Whether to create a movies library (TMDB agent).
+    required: false
+    default: "false"
+  without_shows:
+    description: Whether to create a TV shows library.
+    required: false
+    default: "false"
+  without_music:
+    description: Whether to create a music library.
+    required: false
+    default: "false"
+  without_photos:
+    description: Whether to create a photos library.
+    required: false
+    default: "false"
+outputs:
+  plex_app_data_path:
+    description: "The path to the app data directory."
+    value: ${{ steps.plex.outputs.app_data_path }}
+  plex_plugin_log_path:
+    description: "The path to the plugin log directory."
+    value: ${{ steps.plex.outputs.plugin_log_path }}
+  plex_plugin_path:
+    description: "The path to the plugin directory."
+    value: ${{ steps.plex.outputs.plugin_path }}
+  plex_token:
+    description: "The Plex token for the server."
+    value: ${{ steps.token.outputs.plex_token }}
+
+runs:
+  using: "composite"
+  steps:
+    - name: Build plist
+      shell: bash
+      working-directory: ${{ github.action_path }}
+      run: |
+        echo "::group::Build plist"
+        python ./scripts/build_plist.py
+        echo "::endgroup::"
+
+    - name: Setup plexhints.bundle
+      shell: bash
+      working-directory: ${{ github.action_path }}
+      run: |
+        echo "::group::Setup plexhints.bundle"
+        mkdir -p ./plexhints.bundle/Contents
+
+        # put Contents directory in bundle
+        cp -r ./Contents/. ./plexhints.bundle/Contents
+        echo "::endgroup::"
+
+    - name: Install Python Requirements
+      shell: bash
+      working-directory: ${{ github.action_path }}
+      run: |
+        echo "::group::Install Python Requirements"
+        # install the plexhints library
+        python -m pip install -e .
+
+        # install requirements required for this action to complete
+        python -m pip install -r requirements-action.txt
+        echo "::endgroup::"
+
+    - name: Update Plex registry settings
+      if: ${{ runner.os == 'Windows' }}
+      shell: pwsh
+      working-directory: ${{ github.action_path }}
+      run: |
+        echo "::group::Update Plex registry settings"
+        # starting with pmps 1.32.2 servers must be claimed... disable that
+        # https://forums.plex.tv/t/new-claiming-requirement-for-windows/839096
+        REG ADD "HKCU\Software\Plex, Inc.\Plex Media Server" /v enableLocalSecurity /t REG_DWORD /d 0 /f
```

```suggestion
        REG ADD "HKCU\Software\Plex, Inc.\Plex Media Server" //v enableLocalSecurity //t REG_DWORD //d 0 //f
```

With this syntax can probably put this in the bash section.